src/coreclr/src/System.Private.CoreLib/src/System/Threading/ThreadPool.CoreCLR.cs

```diff
@@ -287,7 +287,7 @@ bool compressStack
             }
             else
             {
```

Comment:
Unrelated nit: you can remove "// throws RegisterWaitException" throughout. this type apparently does not exist -- the comment probably should have been removed before 1.0 release.

Reply:
sure